src/main/java/frc/robot/RobotMap.java

```diff
@@ -111,7 +111,8 @@
   public static final boolean useRemoteImu = false;
 
   public enum RobotId {
-    MINIBOT, ROBOT_2018, ROBOT_2019
+    MINIBOT, COMPETITION_2_2020, COMPETITION_1_2020
```

Comment:
Lets go with PracticeBot and CompetetionBot

Reply:
I updated them to KITBOT and ROBOT_2020